### PR TITLE
Add a couple Gamelab Jr blocks for Kiki

### PR DESCRIPTION
**Pull request by balderdash**
screenshot from 2018-03-06 11-40-02

### CODE REVIEW

File: apps/src/gamelab/blocks.js [hunk 1]
```diff
@@ -211,5 +211,34 @@ export default {
         { name: 'COLOR', type: blockly.BlockValueType.COLOUR },
       ],
     });
+
+    createJsWrapperBlock({
```
Comment (Hamms) [inline apps/src/gamelab/blocks.js:215]: I'm loving this helper

File: apps/src/gamelab/blocks.js [hunk 2]
```diff
@@ -211,5 +211,34 @@ export default {
...
+      blockText: 'move {THIS} to the {POSITION} position',
+      args: [
+        {
+          name: 'POSITION',
+          options: [
+            ['top left', '"top left"'],
+            ['top center', '"top center"'],
+            ['bottom right', '"bottom right"'],
+            ['center right', '"center right"'],
+            ['random', '"random"'],
```
Comment (joshlory) [inline apps/src/gamelab/blocks.js]: Nooooo 😭

Comment (Hamms) [reply apps/src/gamelab/blocks.js]: ?

Comment (balderdash) [reply apps/src/gamelab/blocks.js]: Haha, Josh is not a fan of this dropdown. We'd like to build a better alternative for setting sprite locations (e.g. dragging labeled pins onto the playspace), but I'd just like to get this out for Kiki to prototype with.

File: apps/src/gamelab/GameLabJr.interpreted.js [hunk 3]
```diff
@@ -184,12 +184,27 @@ function makeNewSprite(animation, x, y) {
...
       sprite.y = 200;
+    } else if (position === "bottom left") {
+      sprite.x = 50;
+      sprite.y = 350;
+    } else if (position === "bottom center") {
+      sprite.x = 200;
+      sprite.y = 350;
+    } else if (position === "bottom right") {
+      sprite.x = 350;
+      sprite.y = 350;
```
Comment (joshlory) [inline apps/src/gamelab/GameLabJr.interpreted.js]: I wonder if it's cleaner to have the x and y location as the dropdown value, so the library doesn't need any knowledge of "top left" etc.

File: apps/src/gamelab/blocks.js (hunk 2 above)
Comment (joshlory) [reply apps/src/gamelab/blocks.js]: Yep 😄.  No objections to adding this for prototyping, but I really hope we don't ship with this.

We want students to be thinking with more precision than 9 pre-set locations.  If I can set a sprite to `left-center`, how do I set it to halfway between `left-center` and `left-top`?

**Pull request merged**